### PR TITLE
Fix autoprefixer issue

### PR DESCRIPTION
**Pull request by manishsongirkar**
This PR contain,

1. Fix autoprefixer issue, now compile css with browser prefixes
2. Remove font family and inherit fonts from parent element

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,5 +45,13 @@
 			"<rootDir>/test/unit/setup-blocks.js",
 			"<rootDir>/test/unit/setup-wp-aliases.js"
 		]
-	}
+	},
+	"dependencies": {
+		"autoprefixer": "^8.3.0"
```
Comment (sayedtaqui) [inline package.json]: `dependencies` looks like a weird place for it instead of `devDependencies` is it mentioned on their doc?

Comment (manishsongirkar) [reply package.json]: Okay, my bad. It's also working fine with `devDependencies`.
Updated in latest commit.

**Comment by sayedtaqui** [comment]
@manishsongirkar Can you resolve conflict?

**Comment by manishsongirkar** [comment]
@sayedtaqui Conflicts resolved.

**Pull request merged**